edit-post/assets/stylesheets/_z-index.scss

```diff
@@ -21,6 +20,7 @@ $z-layers: (
 	'.components-popover__close': 5,
 	'.editor-block-list__insertion-point': 5,
 	'.editor-inserter-with-shortcuts': 5,
+	'.editor-warning': 6,
```

Comment:
Are we increasing the `z-index` for the block warning specifically, and if so, will it have negative consequences on any other "EditorWarnings" present in the application? It was meant to be made separate for general reusability, and thus we should be aware of cascading effects on other usage. Alternatively, we could create a separate `z-index` for the specific block warning.